tests/pos/t0273.scala

```diff
@@ -2,6 +2,6 @@ class A
 
 object Test {
 def a = () => ()
-def a[T] = (p:A) => ()
+def a[T] = (p:A) => ()              // error
```

Comment:
is this a pos test or a neg test. 
If some option is needed to make test fail, please specify it.